Makefile

```diff
@@ -39,8 +42,16 @@ all: test build
 
 test: clean cover.out
 
-unit:
-	go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
+unit: kubebuilder
+	$(KUBEBUILDER_ENV) go test $(MOD_FLAGS) $(SPECIFIC_UNIT_TEST) -v -race -count=1 ./pkg/...
+
+# Install kubebuilder if not found
+KUBEBUILDER_DIR := /tmp/kubebuilder_2.3.1_$(OS)_$(ARCH)/bin 
```

Comment:
Can do something like`TMPDIR := $(shell mktemp -d)` instead?

Reply:
I would like to reuse this locally so we don't need to download each time.